### PR TITLE
add tooltip for search syntax

### PR DESCRIPTION
**Pull request by attwad**
image

For #679

### CODE REVIEW

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 1]
```diff
@@ -140,6 +147,79 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
...
+                                                'admin1',
+                                                'admin2',
+                                                'admin3',
+                                                'place',
+                                                'location name',
+                                                'pathogen name',
+                                                'source url',
+                                            ].join(', ')}
+                                            <h5>Keywords search</h5>
+                                            Example:
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:178]: Nit: add `{' '}` after Example:

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 2]
```diff
@@ -140,6 +147,79 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
...
+                                            <i>
+                                                curator:foo@bar.com,fez@meh.org
+                                                country:Japan gender:female
+                                                occupation:"healthcare worker"
+                                            </i>
+                                            values are OR'ed for the same
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:184]: Same here before values

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 1 above)
Comment (attwad) [reply verification/curator-service/ui/src/components/LinelistTable.tsx]: done

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 2 above)
Comment (attwad) [reply verification/curator-service/ui/src/components/LinelistTable.tsx]: done, added a br

**Pull request merged**